README.md

```diff
@@ -3,8 +3,12 @@
 This repository backs Jenkins jobs on a couple of Jenkins masters.
 
 ## Jenkins pipeline definitions under `jobs/`
- 
-An internal [Continuous Infrastructure Jenkins instance](https://atomic-e2e-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/) indexes Jenkinsfiles in the branches of this repository, which are generated from the Jenkinsfiles that live under the `jobs/` directory. Jobs under the `jobs/build/` directory are indexed at the [`aos-cd-builds`](https://atomic-e2e-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/aos-cd-builds/) grouping, while the jobs under `jobs/cluster/` are indexed at the general [`aos-cd`](https://atomic-e2e-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/aos-cd/) grouping. A quick synopsis of these indexed jobs is as follows:
+
+An internal [Continuous Infrastructure Jenkins instance](https://atomic-e2e-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/) indexes Jenkinsfiles in the branches of this repository, which are generated from the Jenkinsfiles that live under the `jobs/` directory. The job responsible for generating, updating and removing the branches can be found in the [`Jenkinsfile`](Jenkinsfile) at the root directory. The builds are currently configured to be executed periodically, but can be manually triggered in [jenkins](https://atomic-e2e-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/aos-cd-master/job/master/).
+
+The scripts that operate on the branches are [`pruner.py`](aos_cd_jobs/pruner.py), which removes branches for jobs that no longer exist, and [`updater.py`](aos_cd_jobs/updater.py), which creates/updates branches for existing jobs. A "job" is any directory under the `jobs/` directory which contains a `Jenkinsfile`.  Every branch is an orphan (doesn't contain any history) and its contents are the contents of the `master` branch with the corresponding directory under `jobs/` copied to the root directory and the `jobs/` directory removed.
```

Comment:
Also, do a simple (snipped) `tree` output of the `master` content and branch content here to show what you mean in the last sentence.